src/Compatibility/Microsoft.DotNet.PackageValidation/PackageValidationLogger.cs

```diff
@@ -9,81 +9,21 @@ namespace Microsoft.DotNet.PackageValidation
 {
     public class PackageValidationLogger : ILogger
     {
-        public void Log(LogLevel level, string data)
-        {
-            switch (level)
-            {
-                case LogLevel.Debug:
-                    LogDebug(data);
-                    break;
-                case LogLevel.Error:
-                    LogError(data);
-                    break;
-                case LogLevel.Information:
-                    LogInformation(data);
-                    break;
-                case LogLevel.Minimal:
-                    LogMinimal(data);
-                    break;
-                case LogLevel.Verbose:
-                    LogVerbose(data);
-                    break;
-                case LogLevel.Warning:
-                    LogWarning(data);
-                    break;
-            }
-        }
-
-        public void Log(ILogMessage message)
-        {
-            Log(message.Level, message.Message);
-        }
-
-        public Task LogAsync(LogLevel level, string data)
-        {
-            Log(level, data);
-            return Task.FromResult(0);
-        }
-
-        public Task LogAsync(ILogMessage message)
-        {
-            Log(message);
-            return Task.FromResult(0);
-        }
-
-        public void LogDebug(string data)
-        {
-            Console.WriteLine($"[DEBUG] {data}");
-        }
-
+        public void Log(LogLevel level, string data) => throw new NotImplementedException();
+        public void Log(ILogMessage message) => throw new NotImplementedException();
+        public Task LogAsync(LogLevel level, string data) => throw new NotImplementedException();
+        public Task LogAsync(ILogMessage message) => throw new NotImplementedException();
+        public void LogDebug(string data) => throw new NotImplementedException();
+        public void LogInformation(string data) => throw new NotImplementedException();
+        public void LogInformationSummary(string data) => throw new NotImplementedException();
+        public void LogMinimal(string data) => throw new NotImplementedException();
+        public void LogVerbose(string data) => throw new NotImplementedException();
+        public void LogWarning(string data) => throw new NotImplementedException();
         public void LogError(string data)
```

Comment:
Curious question, why not use the MSBuild logger that already handles coloring and all that?

Reply:
its an internal class that lives in the arcade. All the loggers in the sdk repo are not that appropiate. eg the errors need to start with ".net sdk error"

Reply:
As noted offline we should use an MSBuild logger rather than logging to the console ourselves.